geode-protobuf/src/main/proto/basicTypes.proto

```diff
@@ -62,4 +62,14 @@ message Region {
 
 message Server {
     string url = 1;
-}
\ No newline at end of file
+}
+
+message Error {
```

Comment:
These are the same fields as ErrorResponse and KeyedErrorResponse in the outstanding PutAll pull request, can you comment in that PR to change the names if you prefer these?